scripts/wasmd/generate_template.sh

```diff
@@ -29,37 +32,27 @@ if [ ! -x "$SCRIPT_DIR/template/.wasmd/config/gentx" ]; then
   sudo chown -R "$(id -u):$(id -g)" "$SCRIPT_DIR/template"
 fi
 
-function inline_jq() {
-  IN_OUT_PATH="$1"
-  shift
-  TMP_DIR=$(mktemp -d "${TMPDIR:-/tmp}/inline_jq.XXXXXXXXX")
-  TMP_FILE="$TMP_DIR/$(basename "$IN_OUT_PATH")"
-  jq "$@" <"$IN_OUT_PATH" >"$TMP_FILE"
-  if ! mv "$TMP_FILE" "$IN_OUT_PATH"; then
-    echo >&2 "Temp file '$TMP_FILE' could not be deleted. If it contains sensitive data, you might want to delete it manually."
-    exit 3
-  fi
-}
-
 (
   cd "$SCRIPT_DIR"
-  # Sort genesis
-  inline_jq "template/.wasmd/config/genesis.json" -S
+  # so weird, but found I needed the -M flag after lots of debugging odd error messages
+  # happening when redirecting stdout
+  jq -S -M . < "template/.wasmd/config/genesis.json" > genesis.tmp
+  mv genesis.tmp "template/.wasmd/config/genesis.json"
 
   # Custom settings in config.toml
-  sed -i "" \
+  sed -i"" \
     -e 's/^cors_allowed_origins =.*$/cors_allowed_origins = ["*"]/' \
-    -e 's/^timeout_propose =.*$/timeout_propose = "300ms"/' \
+    -e 's/^timeout_propose =.*$/timeout_propose = "100ms"/' \
     -e 's/^timeout_propose_delta =.*$/timeout_propose_delta = "100ms"/' \
-    -e 's/^timeout_prevote =.*$/timeout_prevote = "300ms"/' \
+    -e 's/^timeout_prevote =.*$/timeout_prevote = "100ms"/' \
     -e 's/^timeout_prevote_delta =.*$/timeout_prevote_delta = "100ms"/' \
-    -e 's/^timeout_precommit =.*$/timeout_precommit = "300ms"/' \
+    -e 's/^timeout_precommit =.*$/timeout_precommit = "100ms"/' \
     -e 's/^timeout_precommit_delta =.*$/timeout_precommit_delta = "100ms"/' \
-    -e 's/^timeout_commit =.*$/timeout_commit = "1s"/' \
+    -e 's/^timeout_commit =.*$/timeout_commit = "200ms"/' \
     "template/.wasmd/config/config.toml"
 
   # Custom settings app.toml
-  sed -i "" \
+  sed -i"" \
```

Comment:
Are these from `shfmt`?

Reply:
No. The previous way only works on osx and not linux.

I had this issue before. Linux wants `sed -i 's/...'`. OSX wants `sed -i "" 's/...'`. The only way that works on both is `sed -i"" 's/...'`